src/core-sdk-tasks/JsonPropertyParser.cs

```diff
@@ -0,0 +1,61 @@
+using System;
```

Comment:
I noticed that most tasks have a copyright header but not all do. Should I add one here and make a separate PR to add the header to those that don't have it?

Reply:
Yeah, we should make sure we have proper headers in all the source files, at least the .cs ones I think